external_deps/build.sh

```diff
@@ -903,40 +921,78 @@ setup_linux-arm64-default() {
 	common_setup linux aarch64-unknown-linux-gnu
 }
 
+base_windows_amd64_msvc_packages='pkgconfig zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports genlib'
+all_windows_amd64_msvc_packages="${base_windows_amd64_msvc_packages}"
+
+base_windows_i686_msvc_packages="${base_windows_amd64_msvc_packages}"
+all_windows_i686_msvc_packages="${base_windows_amd64_msvc_packages}"
+
+base_windows_amd64_mingw_packages='zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports'
+all_mingw_packages="${base_windows_amd64_mingw_packages}"
+
+base_windows_i686_mingw_packages="${base_windows_amd64_mingw_packages}"
+all_windows_i686_mingw_packages="${base_windows_amd64_mingw_packages}"
+
+base_macos_amd64_default_packages='pkgconfig nasm gmp nettle sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports'
+all_macos_amd64_default_packages="${base_macos_amd64_default_packages}"
+
+base_linux_amd64_default_packages='naclsdk naclports'
+all_linux_amd64_default_packages='zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports'
+
+base_linux_i686_default_packages="${base_linux_amd64_default_packages}"
+all_linux_i686_default_packages="${all_linux_amd64_default_packages}"
+
+base_linux_arm64_default_packages='naclsdk'
+all_linux_arm64_default_packages='zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk'
+
+base_linux_armhf_default_packages="${base_linux_arm64_default_packages}"
+all_linux_armhf_default_packages="${all_linux_arm64_default_packages}"
+
 # Usage
 if [ "${#}" -lt "2" ]; then
-	cat <<-EOF
-	usage: ${0} <platform> <package[s]...>
+	sed -e 's/\\t/  /g' <<-EOF
+	usage: $(basename "${BASH_SOURCE[0]}") <platform> <package[s]...>
 
 	Script to build dependencies for platforms which do not provide them
 
 	Platforms:
-	  windows-i686-msvc windows-amd64-msvc windows-i686-mingw windows-amd64-mingw macos-amd64-default linux-amd64-default linux-i686-default linux-arm64-default linux-armhf-default
+	\twindows-i686-msvc windows-amd64-msvc windows-i686-mingw windows-amd64-mingw macos-amd64-default linux-amd64-default linux-i686-default linux-arm64-default linux-armhf-default
 
 	Packages:
-	  pkgconfig nasm zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports wasisdk wasmtime
+	\tpkgconfig nasm zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports wasisdk wasmtime
 
 	Virtual packages:
-	  install - create a stripped down version of the built packages that CMake can use
-	  package - create a zip/tarball of the dependencies so they can be distributed
-	  wipe - remove products of build process, excepting download cache but INCLUDING installed files. Must be last
+	\tbase — build base packages for the platform (the ones to be packaged)
```

Comment:
To clarify, "base" means the packages built when creating the pre-built binaries hosted on unvanquished.net, and "all" means all supported packages that can possibly be involved in building Daemon and/or Unvanquished?

Reply:
Yes.

Reply:
I rewrote the sentences.